src/Symfony/Bridge/PhpUnit/Compat/Util/Blacklist.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\PhpUnit\Compat\Util;
+
+if (class_exists(PHPUnit\Util\Blacklist::class)) {
+    class Blacklist extends \PHPUnit\Util\Blacklist
+    {}
+} else {
+    class Blacklist extends \PHPUnit_Util_Blacklist
```

Comment:
there is no reason to create this class. We need compat classes only when extending them. Otherwise, we can do the class_exists check around the usage of the class, which is much more readable